profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestAccount.java

```diff
@@ -551,4 +555,37 @@ public void testGetAccountAuditLogs() throws Exception {
         assertEquals(auditLogsJson.get(3).getObjectId(), accountCustomFields.get(2).getCustomFieldId());
     }
 
+    @Test(groups = "slow", description = "retrieve account logs")
+    public void testGetAccountAuditLogsWithHistory() throws Exception {
+        final Account accountJson = createAccount();
+        assertNotNull(accountJson);
+
+        // Update Account
+        final Account newInput = new Account(accountJson.getAccountId(),
+                                             "zozo", 4, accountJson.getExternalKey(), "rr@google.com", 18,
+                                             "USD", null, false, null, null, "UTC",
+                                             "bl1", "bh2", "", "", "ca", "San Francisco", "usa", "en", "415-255-2991",
+                                             "notes", false, false, null, null);
+
+        final Account updatedAccount = killBillClient.updateAccount(newInput, requestOptions);
+
+
+        final List<AuditLog> auditLogWithHistories = killBillClient.getAccountAuditLogsWithHistory(accountJson.getAccountId());
+        assertEquals(auditLogWithHistories.size(), 2);
+        assertEquals(auditLogWithHistories.get(0).getChangeType(), ChangeType.INSERT.toString());
+        assertEquals(auditLogWithHistories.get(0).getObjectType(), ObjectType.ACCOUNT);
+        assertEquals(auditLogWithHistories.get(0).getObjectId(), accountJson.getAccountId());
+
+        final LinkedHashMap history1 = (LinkedHashMap) auditLogWithHistories.get(0).getHistory();
```

Comment:
@sbrossie Curious how it will work with the new code (would be nice if `AuditLog<E>` can be generated).